rollup.config.mjs

```diff
@@ -0,0 +1,23 @@
+import terser from '@rollup/plugin-terser';
+import fs from 'fs/promises';
+import { defineConfig } from 'rollup';
+
+const pkg = JSON.parse(await fs.readFile('package.json', 'utf8'));
+
+const banner = '// https://github.com/farzher/fuzzysort v' + pkg.version;
+const exports = 'named';
+
+export default defineConfig([
+  {
+    input: 'src/fuzzysort.js',
+    output: { banner, exports, file: pkg.browser, format: 'umd', name: 'fuzzysort' },
```

Comment:
while you're at it I suggest also enabling source map output to help debugging.

Reply:
For just the minified file? Or all? The sourcemap is quite big to be checked in IMO

Reply:
I don't mean check in, but generate in build. At least there's no source map comment at the end of the fuzzsort.min.js file.

Reply:
mm, maybe a compromise is to add flag to generate .map files but don't check them in? With reasonable assumption users will run a real build themselves. 
And change back to the non-minified version in the test.html for the quick tests.

Reply:
Done. I've enabled sourcemaps and added the maps to the gitignore. Adding the non-minified version for the tests is a bit tricky since the non-minified versions are ESM and CJS, not IIFE format